tests/README.md

```diff
@@ -22,6 +22,17 @@ Steps:
 
 For more information, refer to the [docker compose documentation](https://docs.docker.com/compose/)
 
+
+# Testing Profil from PostgreSQL database table
```

Comment:
Just to clarify, I didn't remove this. It has been moved.

It was moved in the QGIS project file in the abstract, so it can be displayed in the main Lizmap Web Client application.
Someone visiting https://demo.lizmap.com/lizmap/?repository=modules (with enough user permissions for now) could try and test this setting. 

https://github.com/arno974/lizmap-altiProfil/blob/master/tests/lizmap/instances/altiprofil/altiprofil.qgs#L1389

and

https://github.com/arno974/lizmap-altiProfil/blob/master/tests/lizmap/instances/altiprofil/media/js/altiprofil/open.js